specs/bep-v1.rst

```diff
@@ -196,331 +179,163 @@ For C=1:
+.. Documentation note: the first time a field is mentioned it is put in
+   **bold text**. We use the space separated names in running text and
+   snake_case in the protocol buffer schema.
 
-The **Flags** field contains flags that affect the behavior of the folder. The
-folder Flags field contains the following single bit flags:
+The **folders** field contains the list of folders that will be synchronized
+over the current connection.
 
-::
+Fields (Folder Message)
+~~~~~~~~~~~~~~~~~~~~~~~
 
-     0                   1                   2                   3
-     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
-    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-    |                           Reserved                    |T|D|P|R|
-    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+The **id** field contains the folder ID, which is the unique identified of
+the folder.
 
-:Bit 31 ("R", Read Only):
-    is set for folders that the device will accept no updates from the network
-    for.
+The **label** field contains the folder label, as human readable name for
+the folder.
 
-:Bit 30 ("P", Ignore Permissions):
-    is set for folders that the device will not accept or announce file
-    permissions for.
+The **read only** field is set for folders that the device will accept no
+updates from the network for.
 
-:Bit 29 ("D", Ignore Deletes):
-    is set for folders that the device will ignore deletes for.
+The **ignore permissions** field is set for folders that the device will not
+accept or announce file permissions for.
 
-:Bit 28 ("T", Disable Temporary Indexes):
-    is set for folders that will not dispatch and do not wish to receive
-    progress updates about partially downloaded files via DownloadProgress
-	messages.
+The **ignore delete** field is set for folders that the device will ignore
+deletes for.
 
-The **Options** field contains a list of options that apply to the folder.
+The **disable temp indexes** field is set for folders that will not dispatch
+and do not wish to receive progress updates about partially downloaded files
+via Download Progress messages.
 
-Fields (Device Structure)
+The **devices** field is list of devices participating in sharing this
+folder.
+
+Fields (Device Message)
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The device **ID** field is a 32 byte number that uniquely identifies the
+The device **id** field is a 32 byte number that uniquely identifies the
 device. For instance, the reference implementation uses the SHA-256 of the
 device X.509 certificate.
 
-The **Name** field is a human readable name assigned to the described device
+The **name** field is a human readable name assigned to the described device
 by the sending device. It MAY be empty and it need not be unique.
 
-The list of **Addressess** is that used by the sending device to connect to
+The list of **addressess** is that used by the sending device to connect to
```

Comment:
s/addressess/addresses/
And will need to check if this mis-spelling is elsewhere, to do a global change.